src/main.c

```diff
@@ -1683,6 +1683,56 @@ static gboolean service_start(int argc, char **argv)
 	return TRUE;
 }
 
+static gboolean mount_start(int argc, char **argv)
+{
+	g_autofree gchar *bundlelocation = NULL;
+	g_autoptr(RaucBundle) bundle = NULL;
+	GError *error = NULL;
+	gboolean res = FALSE;
+
+	if (argc < 3) {
+		g_printerr("A file name must be provided\n");
+		r_exit_status = 1;
+		return FALSE;
+	}
+
+	if (argc > 3) {
+		g_printerr("Excess argument: %s\n", argv[3]);
+		r_exit_status = 1;
+		goto out;
+	}
+
+	bundlelocation = resolve_bundle_path(argv[2]);
+	if (bundlelocation == NULL)
+		goto out;
+	g_debug("input bundle: %s", bundlelocation);
+
+	res = check_bundle(bundlelocation, &bundle, TRUE, &error);
+	if (!res) {
+		g_printerr("%s\n", error->message);
+		g_clear_error(&error);
+		goto out;
+	}
+
+	g_message("bundle payload size: %"G_GOFFSET_FORMAT, bundle->size);
+
+	res = mount_bundle(bundle, &error);
+	if (!res) {
+		g_printerr("%s\n", error->message);
+		g_clear_error(&error);
+		goto out;
+	}
+
+	g_message("bundle mounted at: %s", bundle->mount_point);
+
+	/* The mount keeps the block device active. */
```

Comment:
How is this comment related to the `exit` call?

Reply:
The device mapper target and loopback devices are configured to remove themselves on close. So we don't need to handle cleanup in RAUC, a simple unmount on the is enough. I'll extend the comment accordingly.

Reply:
I've expanded the comment.